tests/test_all_rules.py

```diff
@@ -639,6 +639,36 @@ def test_deprecated_rules_modified(self):
         if result:
             self.fail(f"Deprecated rules {result} has been modified")
 
+    def test_rule_change_has_updated_date(self):
+        """Test to ensure modified rules have updated_date field updated."""
+
+        rules_path = get_path("rules")
+        rules__bbr_path = get_path("rules_building_block")
```

Comment:
```suggestion
        rules_bbr_path = get_path("rules_building_block")
```

Nit, do we need the additional "_"? If renaming will need to rename elsewhere in the additions as well.